modules/gitops-release-notes-1-4-13.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+:_content-type: REFERENCE
+
+[id="gitops-release-notes-1-4-13_{context}"]
+= Release notes for {gitops-title} 1.4.13
+
+{gitops-title} 1.4.13 is now available on {product-title} 4.7, 4.8, 4.9, 4.10 and 4.11.
+
+[id="fixed-issues-1-4-13_{context}"]
+== Fixed issues
+
+The following issues have been resolved in the current release:
+
+* From {product-title} 4.12, it will be optional to install the console. This fix updates the {gitops-title} Operator to prevent errors with the Operator if the console is not installed. link:https://issues.redhat.com/browse/GITOPS-2354[GITOPS-2354]
```

Comment:
do you need to use future tense here? Or can you simply say, `From {product-title} 4.12, it is optional to install the console.`